### ISSUE TITLE
[Keyboard Navigation- Make code - History]: "2/8/2024 ↑3" button and other controls are not accessible through keyboard.

### ISSUE THREAD

**Issue by Yamini234u** [microsoft/pxt-microbit#5500]
### **User Experience**
Keyboard users will not be able access the 'History ↑2' control if the focus is not moving to the controls. Users might get impacted if it is not accessible with keyboard and will face challenges and get misguided.

**Note:**
User credentials should NOT be included in the bug.

### **Repro Steps:**

1. Launch application URL : https://makecode.microbit.org
2. Home page will open.
3. Navigate "New Project" option and select it.
4.  Editor screen will open.
5. Navigate to "Review, Commit and push local changes to GitHub" button and select it.
6. Navigate to all the controls under History, Release Zone, Extension Zone Section.
7. Navigate to "View Commits" button and "↑3" button will appear
8. Observe and verify "2/8/2024↑3" button accessible through keyboard or not?

### **Actual Result:**
"2/8/2024 ↑3" button and other control are not accessible through keyboard.
**Observation:** 
- On navigating with key focus is moving to "2/8/2024 ↑3" button. 
- Tabindex is not defined for the button

### **Expected result:**
"2/8/2024 ↑3" button and other controls should be accessible through keyboard.
Solution: Focus should move to "2/8/2024 ↑3" button and other controls under "↑3" button.

### **MAS Reference:**
[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA

**DOM details available:**
Yes(Please refer to the attachment)

### **Reference Links:**
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDc3NDY0MzExZDkxNTRlYmRhNzk2YTExYzE5Y2I2N2ZifDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3ODcwNDg4MTQ1NjM2NzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT0%3D&sdata=VHBCSFBzMTdmWExVNTk3MzZlTHdiME4rcFo3d3lyYk5kTTIvRjhLdTN3QT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-chbhardwaj%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.
### **Test Environment:**
- **OS Version:** Windows 11 23H2 (OS Build:22631.3007)
- **Browser:** Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- **Test Matrix**: Chrome + JAWS
- **URL:** https://makecode.microbit.org/.
<img width="965" alt="MAS2 1 1- History ↑3 button and other controls are not accessible through keyboard" src="https://github.com/microsoft/pxt-microbit/assets/117158108/ec4d54de-3088-47db-b6c0-0069c64b4fb1">
<img width="970" alt="MAS2 1 1-DOM- 2-8-2024 ↑3 button and other controls are not accessible through keyboard" src="https://github.com/microsoft/pxt-microbit/assets/117158108/954d4974-0cab-40f1-a83d-be9f2cba4463">

**Comment by Yamini234u** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#A11yAuto;#k4W;#MAS2.1.1;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Feb2024;TabStops
